src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/modules/DashboardManageVariableTable.vue

```diff
@@ -92,17 +92,22 @@ const props = defineProps<Props>();
 const emit = defineEmits<EmitFn>();
 
 const state = reactive({
-    orderedVariables: computed(() => props.order.map((d) => {
-        const currentVariable = {
-            ...props.variables[d],
-            propertyName: d,
-        };
-        if (currentVariable.variable_type === 'MANAGED') return currentVariable;
-        return {
-            ...currentVariable,
-            managable: d,
-        };
-    })),
+    orderedVariables: computed(() => {
+        const convertedVariables = props.order.map((d) => {
+            const currentVariable = {
+                ...props.variables[d],
+                propertyName: d,
+            };
+            if (currentVariable.variable_type === 'MANAGED') return currentVariable;
+            return {
+                ...currentVariable,
+                managable: d,
+            };
+        });
+        if (state.selectedVariableType === 'ALL') return convertedVariables;
```

Comment:
Please consider with making constant about `'MANAGED'` and `'ALL'` if they are used by other files!

Reply:
I'd like to have some discussions about it next time!